spec/sec-properties-of-the-regexp-prototype-object-patch.html

```diff
@@ -91,8 +89,9 @@ <h1>Runtime Semantics: RegExpBuiltinExec ( _R_, _S_ )</h1>
             1. <ins>Append _s_ to _groupNames_.</ins>
           1. <ins>Else,</ins>
             1. <ins>If _groupNames_ is a List, append *undefined* to _groupNames_.</ins>
-        1. <ins>Set _getIndices_.[[MatchIndices]] to _indices_.</ins>
-        1. <ins>Set _getIndices_.[[GroupNames]] to _groupNames_.</ins>
+        1. <ins>Let _indicesArray_ be MakeIndicesArray(_S_, _indices_, _groupNames_).</ins>
+        1. <ins>Perform ! DefinePropertyOrThrow(_A_, `"indices"`, PropertyDescriptor { [[Value]]: _indicesArray_, [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }).</ins>
```

Comment:
Is there a reason `"indices"` is defined using `DefinePropertyOrThrow` rather than just using `CreateDataProperty` like `"groups"`, `"index"`, and `"input"`?

Also, if we are going to just make `"indices"` a regular data property, should we just inline the logic of `MakeIndicesArray` into the existing loop on line 63?

Reply:
actually it seems like the usage of `CreateDataProperty` is incorrect without an assertion that the status is always true; using `! CreateDataPropertyOrThrow` seems like the better choice for all of them?

Reply:
Also, I noticed that while writing test262 tests that the actual behavior in v8 v7.8.137 is that the `indices` property is `{ writable: true, configurable: true, enumerable: true }`.

Reply:
As an own data property, that seems right to me - that's also what https://tc39.es/ecma262/#sec-createdataproperty and https://tc39.es/ecma262/#sec-createdatapropertyorthrow both do.

Reply:
That's why I'm wondering if the line above is incorrect.

Reply:
I believe it is; these should be using CreateDataProperty and asserting that the status is true, or, using `! CreateDataPropertyOrThrow`.

Reply:
Good catch, will fix.